<launch>
  <node pkg="roslink-ba" name="udp_client_roslink_bridge_turtlebot_node" type="roslink_bridge_riae100_WS.py" output="screen"/>

  <param name="ground_station_ip" value='127.0.0.1' />
  <param name="ground_station_port" value='25500' />

  <param name="heartbeat_msg_rate" value="1" />
  <param name="robot_status_msg_rate" value="0.5" />
  <param name="global_motion_msg_rate" value="1" />
  <param name="gps_raw_info_msg_rate" value="0.5" />
  <param name="range_finder_data_msg_rate" value="0.5" />
  <param name="map_location" value="/home/ria/my_map.png" />
  <param name="map_xoffset" value="-10"/>
  <param name="map_yoffset" value="-21" />
  <param name="ROSLing_Image_msg_rate" value="2" />

  <param name="roslink_version" value="1" />
  <param name="ros_version" value="8"/>
  <param name="system_id" value='12'/>
  <param name="robot_name" value="MyRia-E100"/>
  <param name="type" value="0"/>
  <param name="owner_id" value="1"/>
  <param name="key" value="o22Yr"/>
<!--
This launch file is an example for 3D registration using DSO stero reconstruction
  <include file="$(find ria_bringup)/launch/minimal.launch"/>

-->
</launch>
